grails-app/services/de/iteratec/osm/report/external/MetricReportingService.groovy

```diff
@@ -245,7 +244,7 @@ class MetricReportingService {
         log.debug("reporting page csi-values with intervalInMinutes ${intervalInMinutes} for reportingTimestamp: ${reportingTimeStamp}")
 
         def groups = jobGroupService.findCSIGroups().findAll {
-            it.graphiteServers.size() > 0 && it.graphiteServers.any { server -> server.reportCsiAggregationsToGraphiteServer }
+            it.resultGraphiteServers.size() > 0 && it.resultGraphiteServers.any { server -> server.reportCsiAggregationsToGraphiteServer }
```

Comment:
I know you just renamed the variables, but while we are at it you could simplify that with some groovy truthiness, where an empty list is false and a so is null
Either:
it.resultGraphiteServers && it.resultGraphiteServers.any{...}
or even better:
it.resultGraphiteServers?.any{...}
In both cases you would also avoid a possible NPE